ch_tools/monrun_checks/ch_resetup_state.py

```diff
@@ -1,60 +1,44 @@
 import json
 import os
-import socket
 import subprocess
 from typing import Any
 
 import click
 import psutil
-import requests
 
-from ch_tools.common.clickhouse.client.clickhouse_client import clickhouse_credentials
 from ch_tools.common.clickhouse.config.path import CLICKHOUSE_RESETUP_CONFIG_PATH
 from ch_tools.common.result import CRIT, OK, Result
 from ch_tools.monrun_checks.exceptions import die
 
 
+# TODO: delete unused ssl and ca_bundle options after some time
 @click.command("resetup-state")
-@click.option("-p", "--port", "port", type=int, help="ClickHouse HTTP(S) port to use.")
-@click.option("-s", "--ssl", "ssl", is_flag=True, help="Use HTTPS rather than HTTP.")
-@click.option("--ca_bundle", "ca_bundle", help="Path to CA bundle to use.")
-@click.pass_context
-def resetup_state_command(
-    ctx: click.Context, port: int, ssl: bool, ca_bundle: Any
-) -> Any:
+@click.option("-s", "--ssl", "_ssl", is_flag=True, help="Use HTTPS rather than HTTP.")
+@click.option("--ca_bundle", "_ca_bundle", help="Path to CA bundle to use.")
+def resetup_state_command(_ssl: bool, _ca_bundle: Any) -> Any:
     """
     Check state of resetup process.
     """
 
-    check_repsync_running()
     check_resetup_running()
     check_resetup_required()
 
-    host = socket.getfqdn()
-    if request(ctx, host, port, ssl, ca_bundle):
-        return Result(CRIT, "ClickHouse is listening on ports reserved for resetup")
-
     if os.path.isfile(CLICKHOUSE_RESETUP_CONFIG_PATH):
-        return Result(CRIT, "Detected resetup config, but ch-backup is not running")
+        return Result(
+            CRIT, "Detected resetup config, but couldn't find running resetup process"
+        )
 
     return Result(OK)
 
 
 def check_resetup_running() -> None:
     """
-    Check for currently running `ch-backup restore-schema`
+    Check for currently running resetup
     """
     for proc in psutil.process_iter():
         if {"/usr/bin/ch-backup", "restore-schema"}.issubset(proc.cmdline()):
             die(0, "resetup is running")
```

Comment:
Let's add ` (restore schema)` to the message for consistency with the case below.

Reply:
agreed, done
https://github.com/yandex/ch-tools/pull/416/commits/dfc9085ee6f28fbc0ba09f4c1b1716f50a1c9358